client/src/components/Header/Header.js

```diff
@@ -1,15 +1,19 @@
 import React from "react";
+import styled from "styled-components";
 
-import "./header.css";
+const HeaderStyled = styled.header`
+  font-family: "Orbitron", Arial, sans-serif;
```

Comment:
you probably don't need the font-family here, it's already in globalStyles

Reply:
Thanks for the hint